### ISSUE TITLE
Add WebDav sync? 

### ISSUE THREAD

**Issue by TobiasDev** [moneymanagerex/android-money-manager-ex#1243]
I switched from Dropbox to NextCloud as my current cloud provider. I would love if I could sync Money Manager with my NextCloud instance instead of Dropbox. You can sync very easily to NextCloud with WebDav sync.

**Comment by MisterY** [comment]
Great idea. If you submit a pull request with the implementation it will be accepted.

**Comment by Whisprin** [comment]
If found older similar issues mentioning CloudRail. Are you still using it? @MisterY

**Comment by MisterY** [comment]
Hi! Recently I started using pCloud and added it to the list of providers (#1249). Unfortunately, there is a bug in CloudRail so the files can not be overwritten when uploaded. For this reason I've reverted to using FolderSync. This works without issues so far.

This has a couple of consequences. It shows that supporting cloud synchronization is a major pain for an application that does not specialize in the area. It also shows that specialized apps, even though there are only few and they are paid apps, are doing their job. There is an issue in caching sqlite databases at Android level and this needs to be looked at. 

More and more storage providers are creating their own mobile apps so I'm pretty confident they will do a better job than we can. The users should request their favorite provider to also provide a decent sync app.

Due to these things, I've started marking the open issues with the Sync tag. There are quite a few of them. It seems that implementing #1250 would significantly reduce the number of reported issues in the backlog. 

The level of support the synchronization with constantly evolving and growing services does not seem realistic for a project this small.

So, the short answer would be - no. I'm still not 100% happy with the 3rd-party sync apps but, on the other hand, they relieve me from having to spend endless time supporting users having issues synchronizing with their favorite provider, updating the protocol library (CloudRail or a direct client), responding to requests to add another provider, etc. It looks like eliminating the direct synchronization support is the way forward. I'd like to think of MMEX as a mobile personal finance app (or, rather, an expense tracker) and not much more.
